mobify/platform-scaffold
# [Integration Manager] Global merlins connector

This PR moves global actions to the merlin's connector

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1272
 **Linked PRs**:  #436 

## Changes
- Moved footer actions and parsing to the merlin's connector
- Moved global actions to set the current page and check if offline to execute after the containers fetcher has run
- Moved getCart action to the merlin's connector

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to pages converted to the integration manager (home, PLP, PDP) and pages that have not been converted yet (login, cart, checkout)
- All pages should work as expected
